backend/typescript/models/scheduling.model.ts

```diff
@@ -0,0 +1,38 @@
+import { Column, DataType, Model, Table } from "sequelize-typescript";
+import { Status } from "../types";
+
+@Table({ tableName: "scheduling" })
+export default class Scheduling extends Model {
+  @Column({ type: DataType.INTEGER })
+  id!: number;
+
+  @Column({ type: DataType.INTEGER })
+  donor_id!: string;
+
+  @Column({ type: DataType.STRING })
+  auth_id!: string;
+
+  @Column({ type: DataType.TEXT })
+  description!: string | null;
+
+  @Column({ type: DataType.INTEGER })
+  quantity!: number | null;
+
+  @Column({ type: DataType.DATE })
+  start_time!: Date;
+
+  @Column({ type: DataType.DATE })
+  end_time!: Date;
+
+  @Column({ type: DataType.ENUM("Rejected", "Approved", "Pending") })
+  status!: Status;
+
+  @Column({ type: DataType.BOOLEAN })
+  volunteer_needed!: boolean;
+
+  @Column({ type: DataType.INTEGER })
```

Comment:
is this the volunteer that is allocated for the specific schedule slot?

Reply:
if there's multiple volunteers should this be an array instead?
```TypeScript
volunteer_id!: number[] | null
```

Reply:
do we need more than one volunteer for any slots? @Shehryar21

Reply:
I guess it wouldn't hurt to support it

Reply:
We could support that but I am just thinking that then we probably would want to ask how many volunteers needed as well?

Reply:
Yep, we could change volunteers_needed to an int, and if volunteers aren't needed this would just be 0

Reply:
yup that works. Default value would be 0

Reply:
Updated!